pkg/noobaaaccount/reconciler.go

```diff
@@ -305,6 +305,17 @@ func (r *Reconciler) ReconcileDeletion() error {
 			return fmt.Errorf("failed to delete account %q. got error: %v", r.NooBaaAccount.Name, err)
 		}
 	} else {
+		if exists := util.IsAnnotationPresent(r.NooBaa.GetAnnotations(), "remote-operator"); exists {
```

Comment:
I think it's better to check for a specific value, and not just the existance of the annotation